src/sanscript.schema.json

```diff
@@ -0,0 +1,101 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/sanskrit/sanscript.js/raw/master/src/sanscript.schema.json",
+    "title": "Sanscript Scheme",
+    "description": "A representation of a writing system (Brahmic or roman) that can be used for Sanskrit and other Indic languages",
```

Comment:
Here too, I lowercased "roman" to be consistent with comments. In any case I don't think "roman" is a proper noun here because it does not refer to the city of Rome or the Roman Empire (cf. "roman letters" as opposed to italic letters; this nomenclature is reflected in LaTeX's `\textrm`). [The Wiktionary page for "roman"](https://en.wiktionary.org/wiki/roman#English) includes common noun usages but not this specific one, but anyway "romanization" is not capitalized. My exclusion of punctuation was in accordance with the JSON Schema example and because this is a noun phrase and not a full sentence.

Reply:
I suggested capitalizing because "Roman" because of the term "Roman script." As per the Wiktionary definition of "roman" that you linked:

> (of text, computing) of or related to the Latin alphabet

totally convinces me. And I see what you're saying about it being a noun phrase. Please ignore.